### PR TITLE
Port explanation of Math.log1p to Math.expm1; wording fixes

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Motivation

When I was looking at `expm1`, I had little idea what it's for, until I found the explanation in `log1p`. It would be beneficial to synchronize the two pages. In addition, the notion:

> When you calculate log(1 + x), you should get an answer very close to x, if x is small (that's why these are called 'natural' logarithms)

is incorrect: at least I don't think that's what "natural" logarithm is "natural" about—I could be wrong—so I made it more mathematical.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/math/expm1/index.md [hunk 1]
```diff
@@ -30,14 +28,15 @@ Math.expm1(x)
...
+
+When you calculate <math display="inline"><semantics><msup><mrow><mi mathvariant="normal">e</mi></mrow><mi>x</mi></msup><annotation encoding="TeX">\mathrm{e}^x</annotation></semantics></math> where x is a number very close to 0, you should get an answer very close to 1 + x, because <math display="inline"><semantics><mrow><munder><mo movablelimits="true" form="prefix">lim</mo><mrow><mi>x</mi><mo stretchy="false">→</mo><mn>0</mn></mrow></munder><mfrac><msup><mrow><mi mathvariant="normal">e</mi></mrow><mi>x</mi></msup><mrow><mi>x</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>=</mo><mn>1</mn></mrow><annotation encoding="TeX">\lim_{x \to 0} \frac{\mathrm{e}^x}{x+1} = 1</annotation></semantics></math>. If you calculate `Math.exp(1.1111111111e-15) - 1`, you should get an answer close to `1.1111111111e-15`. Instead, due to the highest significant figure in the result of `Math.exp` being the units digit `1`, the final value ends up being `1.1102230246251565e-15`, with only 3 correct digits. If, instead, you calculate `Math.exp1m(1.1111111111e-15)`, you will get a much more accurate answer `1.1111111111000007e-15`, with 11 correct digits of precision.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/math/expm1/index.md:37]: Maintaining these chunks of MathML is a pain. The font/styling is also very weird. Do we have plans to onboard KaTeX/MathJax?

Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/math/expm1/index.md:37]: No, not for the moment.

File: files/en-us/web/javascript/reference/global_objects/math/log1p/index.md [hunk 2]
```diff
@@ -50,32 +49,17 @@ Math.log1p(x)
 
 ### Return value
 
-The natural logarithm (base {{jsxref("Math.E", "e")}}) of **1** plus the
-given number. If the number is less than **-1**, {{jsxref("NaN")}} is
-returned.
+The natural logarithm (base {{jsxref("Math.E", "e")}}) of **1** plus the given number. If the number is less than **-1**, {{jsxref("NaN")}} is returned.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/math/log1p/index.md:52]:
```suggestion
The natural logarithm (base {{jsxref("Math.E", "e")}}) of `1` plus the given number. If the number is less than `-1`, {{jsxref("NaN")}} is returned.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A nit and we are good to go.

File: files/en-us/web/javascript/reference/global_objects/math/expm1/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/math/expm1/index.md:37]: OK; what about at least having a better stylesheet for MathML? I guess that would be in yari... right?

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**